Makefile

```diff
@@ -33,11 +33,9 @@ APPS1 = logmanager ledmanager downloader verifier client zedrouter domainmgr ide
 
 SCRIPTS = \
```

Comment:
I could not figure out where `$(SCRIPTS)` is used elsewhere in this `Makefile`. Do other makefiles `include` this one?

Reply:
Good catch